command/command.go

```diff
@@ -28,16 +29,51 @@ import (
 
 var verbose = false
 
+type LogLevel string
+
+const (
+	INFO LogLevel = "INFO"
+	WARN LogLevel = "WARN"
+	ERROR LogLevel = "ERROR"
+)
+
 func printVerboseMessage(message string) {
 	if verbose {
 		fmt.Println(message)
 	}
+	logToFile(INFO, message)
 	logger := client.Config{}.Logger
 	if logger != nil {
 		logger.Debug(message)
 	}
 }
 
+/*
+This method Print the message to console and Logfile based on corresponding Log Level
+ */
+func printMessage(logLevel LogLevel, message string) {
+	log.Println(string(logLevel) + " : " + message)
+	logToFile(logLevel, message)
+}
```

Comment:
Why we can't use printMessage instead of printVerboseMessage?

Reply:
Made the changes and removed printVerboseMessage. Create a new Log Level Debug. which will be used verbose